matisse/src/main/java/com/zhihu/matisse/internal/ui/BasePreviewActivity.java

```diff
@@ -182,6 +182,9 @@ private void updateApplyButton() {
         if (selectedCount == 0) {
             mButtonApply.setText(R.string.button_apply_disable);
             mButtonApply.setEnabled(false);
+        } else if (selectedCount == 1 && mSpec.singleSelectionModeEnabled()) {
+            mButtonApply.setText(R.string.button_apply_disable);
```

Comment:
I think we should rename 'R.string.button_apply_disable' to 'R.string.button_apply_default'.